config/base/derive/src/documented.rs

```diff
@@ -47,17 +47,30 @@ fn impl_get_doc_recursive(docs: Vec<LitStr>, ast: &StructWithFields) -> proc_mac
         .zip(docs)
         .map(|(field, documentation)| {
             let inner_thing = field.has_inner;
+            let option_thing = field.has_option;
             let ty = &field.ty;
             let ident = &field.ident;
-            if inner_thing {
+            let documented_trait = quote! { iroha_config_base::proxy::Documented };
```

Comment:
i would prefer strong typing here

```suggestion
            let documented_trait: syn::Path = syn::parse_quote!{ iroha_config_base::proxy::Documented };
```